devel/valgrind/pkg-descr

```diff
@@ -0,0 +1,3 @@
+Valgrind is a flexible program for debugging and profiling Linux executables.
```

Comment:
Should this description be adapted to DragonFly maybe ?

Reply:
Changed.